src/backend/routers/chat.py

```diff
@@ -69,6 +73,7 @@ async def chat_stream(
                 session=session,
                 conversation_id=conversation_id,
                 user_id=user_id,
+                trace_id=trace_id,
```

Comment:
Should we have a session type object that has all the session info including the DBsession above?

Reply:
I added the minimum for now since Tanzim is OOO. Once he's back, I'll catch up with him and check what else we should add.